lib/json/test/valid/string_in_object.json

```diff
@@ -1 +1,3 @@
-{"key":"hello this is a string"}
\ No newline at end of file
+{
```

Comment:
I understand that. However, the JSON implementation is an upstream of https://github.com/dominictarr/JSON.sh and should not be modified.